decidim-budgets/app/packs/stylesheets/budgets.scss

```
@@ -130,7 +136,7 @@
   }
 
   &__list--header {
-    @apply grid grid-cols-2;
+    @apply flex items-center -ml-2 sm:ml-0;
```

```suggestion
    @apply flex items-center md:-ml-2 ml-0;
```

feedback: prefer mobile first, and then setup for larger screen.